test/e2e/chaos_test.go

```diff
@@ -38,8 +39,10 @@ func TestHAEtcdChaos(t *testing.T) {
 	clusterOpts.NodePoolReplicas = 0
 
 	e2eutil.NewHypershiftTest(t, ctx, func(t *testing.T, mgtClient crclient.Client, hostedCluster *hyperv1.HostedCluster) {
+		t.Run("SingleMemberRecovery", testSingleMemberRecovery(ctx, mgtClient, hostedCluster))
```

Comment:
nit: I would make explicit in the name this test case is for data loss e.g. `SingleMemberDataLossRecovery`